documentation/modules/upgrading/proc-upgrade-cruise-control-api-version.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// assembly-upgrade-resources.adoc
+
+[id='proc-upgrade-cruise-control-api-version-{context}']
+= Upgrading Cruise Control to support {ApiVersion}
+
+.Prerequisites
+
+* A Cluster Operator supporting the `{ApiVersion}` API version is up and running.
+
+.Procedure
+Execute the following steps for each `Kafka.spec.cruiseControl` configuration in your Kafka cluster.
+
+. Update the `Kafka` resource in an editor.
++
+[source,shell,subs="+quotes,attributes"]
+----
+kubectl edit kafka _my-cluster_
```

Comment:
General: `kubectl edit kafka _NAME-OF-KAFKA-CLUSTER_`

Reply:
Not sure what I was thinking there! Updated to capitals throughout, but without "name of" for brevity.

`kubectl edit kafka _KAFKA-CLUSTER_`